.github/workflows/docker_rebuild.yml

```diff
@@ -27,7 +27,7 @@ permissions: {}
 
 jobs:
   image:
-    if: github.repository == 'pmem/pmdk'
```

Comment:
Can we have just verification if it is the primary repo, not a fork?
Example from CP:
```
jobs:
  conditional-job:
    runs-on: ubuntu-latest
    steps:
      - name: Run only if this is the primary repository
        if: ${{ github.event.repository.fork == false }}
        run: echo "This is the primary repository."
```